whist/server/services/game_db_service.py

```diff
@@ -24,9 +24,9 @@ def __new__(cls):
 
     # pylint: disable=too-many-arguments
     @classmethod
-    def create_with_pwd(cls, game_name: str, creator: Player,
-                        hashed_password: Optional[str] = None,
-                        min_player: int = 4, max_player: int = 4) -> 'GameInDb':
+    def create_with_pwd(cls, game_name: str, creator: Player, hashed_password: Optional[str] = None,
+                        min_player: Optional[int] = None,
+                        max_player: Optional[int] = None) -> 'GameInDb':
```

Comment:
Why not use the default parameter here? It is less boilerplate and will automatically be documented which is not the case in setting default inside a function.